_data/contributors.yml

```diff
@@ -348,12 +348,18 @@
   languages:
     - vi
 
-- github: notactuallypagemcconnell
```

Comment:
Note: I noticed that @notactuallypagemcconnell appears twice in the list, and previous entry is more complete, so I deleted this one (sorry @notactuallypagemcconnell 😆)